pkg/types/validation/installconfig.go

```diff
@@ -1345,3 +1355,13 @@ func validateReleaseArchitecture(controlPlanePool *types.MachinePool, computePoo
 
 	return allErrs
 }
+
+// supportedSingleNodePlatform indicates if the IPI Installer can be used to install SNO on
+// a platform.
+func supportedSingleNodePlatform(isSingleNodeOpenShifti bool, p *types.Platform) bool {
```

Comment:
@zaneb , @cybertron Are the supported platforms accurate here?

Reply:
I don't know if SNO is supported on platform `External` . If so, does it need a valid `bootstrapInPlace` config?

Reply:
I don't think it is supported on platform:external